homepage/config/Dockerfile

```diff
@@ -0,0 +1,6 @@
+FROM debian:stable-slim
+WORKDIR /test
+VOLUME /app/config
+RUN apt-get -y update && apt-get install -y gettext
+#COPY services.yaml bookamrks.yaml docker.yaml settings.yaml widgets.yaml ./
+COPY services.yaml /test/
```

Comment:
You're copying to /test but setup.sh copies from /config

Reply:
To be more clear: Dockerfile is not copying the services.yaml file to anywhere. After I run `make install` and it shows no errors, `/var/lib/docker/volumes/homepage_config/_data/` is empty.

Reply:
/test is not a volume, it bakes it into the config container image.